samples/CMakeLists.txt

```diff
@@ -35,6 +35,15 @@ if (CONFIG_SPM)
   endif()
 endif()
 
+if (CONFIG_BUILD_WITH_TFM AND CONFIG_UART_1_NRF_UARTE)
```

Comment:
This is true, though, @anangl , when we build TF-M with SECURE_UART, is that accurate? 
If really so, could we somehow reflect this here?

Reply:
also, is this the right place ?

Isn't this in reality an upstream issue, and should somehow be handled when using TF-M in general (if we can do that.)

As a 1.5 short term fix, I believe this is acceptable.

Reply:
I fully agree this should be solved properly upstream. But the intention was to provide some helper for the 1.5.0 release, to avoid confusion when an application is built with an invalid configuration and there is no indication of it.

@ioannisg I don't know actually how to get here information if TF-M is built with `SECURE_UART1` on. But since this option is currently hard-coded for nRF platforms (or am I wrong @oyvindronningstad?), this should be fine for now as it is, I think.

Reply:
It's hardcoded, yes.

Reply:
If we later make SECURE_UART1 configurable via kconfig, we can update this line.